design-patterns/structural-patterns/src/decorator/SauceSandwich.java

```diff
@@ -0,0 +1,21 @@
+package decorator;
+
+public class SauceSandwich implements Sandwich{
```

Comment:
It is a good code style if you leave a space between the the name and  '{' in classes, methods and thinks like that

Reply:
Fixed.